common/common_target.mk

```diff
@@ -138,7 +140,7 @@ CROSS_ASMFLAGS+=-f elf64
 # Default AR Flags
 ################################################################################
 
-NATIVE_ARFLAGS+=rcs
+NATIVE_ARFLAGS+=-rcs
```

Comment:
Just ran this branch through the gamut of tests on the Windows side, seems fine.